conf/messages.cy

```diff
@@ -1,11 +1,22 @@
-service.name = Penalties
```

Comment:
I assume this file is a placeholder for the welsh translation coming in the future?

Reply:
The whole file should be a direct copy of the messages(.en) file.

So we will pass the messages to the Welsh team for translation once we are happy with the English. 😄

Reply:
Ok cool :)